docker-compose.yml

```diff
@@ -1,22 +1,24 @@
 version: '2'
 
 services:
+  mongodb:
+    image: 'bitnami/mongodb'
+    # volumes:
+    #   - 'mongodb_data:/bitnami'
+
   node:
     tty: true # Enables debugging capabilities when attached to this container.
     image: 'bitnami/node:latest'
-    labels:
-      kompose.service.type: nodeport
     command: sh -c 'npm install && node app.js'
+    environment:
+      APOS_MONGODB_URI: 'mongodb://mongodb:27017'
     ports:
       - 3000:3000
     volumes:
       - .:/app
     depends_on:
       - mongodb
-    environment:
-      APOS_MONGODB_URI: 'mongodb://mongodb:27017'
-  mongodb:
-    tty: true
-    image: 'bitnami/mongodb'
-    ports:
-      - 27017:27017
+
+# volumes:
+#   mongodb_data:
+#     driver: local
```

Comment:
As mentioned in previous comment, I would remove this if it is not going to be used for now.

Reply:
👍